src/components/ftu/ftu.jsx

```diff
@@ -0,0 +1,55 @@
+var React = require('react');
+
+var FTU = React.createClass({
+  mixins: [require('react-intl').IntlMixin],
+  propTypes: {
+    singleMessage: React.PropTypes.string,
+    rightMessage: React.PropTypes.string,
+    leftMessage: React.PropTypes.string
+  },
+  getInitialState: function () {
+    return {
+      isVisible: !this.props.hidden
```

Comment:
Using state here creates a weird fragmented situation where if you change `this.state.isVisible` to false, whatever you were using to set `this.props.hidden` won't get updated. What do you think about passing in a `setVisibility` function or something instead?